src/bert_validator.erl

```diff
@@ -5,9 +5,9 @@
 -compile(export_all).
 
 -define(Valid_Start,     "ErrFields = lists:flatten(
-		  [case {Field, F} of\n\t").
--define(Valid_End(Name),"\n\t_ -> Field
-    end || {Field, F} <- lists:zip(record_info(fields, '"++Name++"'), tl(tuple_to_list(D)))]),").
+		  [case {RecField, F} of\n\t").
+-define(Valid_End(Name),"\n\t_ -> RecField
+    end || {RecField, F} <- lists:zip(record_info(fields, '"++Name++"'), tl(tuple_to_list(D)))]),").
```

Comment:
I would use `___RecField___` or something to avoid more name clashes.
And what on earth is the D this is supposed to always be bound? This is horrible.